### ISSUE TITLE
feature request: add a shortcut to the web interface

### ISSUE THREAD

**Issue by formula-spectre** [mtotschnig/MyExpenses#1292]
it would be nice if i can have a tile in the system quick settings or in the three-dots menu

**Comment by mtotschnig** [comment]
I agree, a tile is a good idea. The idea of making it available from the main screen was already brought up in #1032

**Comment by mtotschnig** [comment]
https://developer.android.com/develop/ui/views/quicksettings-tiles

**Comment by formula-spectre** [comment]
> https://developer.android.com/develop/ui/views/quicksettings-tiles

that is a good start; wish I could contribute but I have very little time and my java/kotlin is rusty..
